crates/bevy_ui/src/render/ui_material_pipeline.rs

```diff
@@ -642,9 +642,11 @@ pub fn queue_ui_material_nodes<M: UiMaterial>(
                 bind_group_data: material.key.clone(),
             },
         );
-        transparent_phase
-            .items
-            .reserve(extracted_uinodes.uinodes.len());
+        if transparent_phase.items.capacity() < extracted_uinodes.uinodes.len() {
+            transparent_phase.items.reserve_exact(
+                transparent_phase.items.capacity() - extracted_uinodes.uinodes.len(),
```

Comment:
This doesn't look right, the capacity the first time the transparent items buffer is filled its capacity will be 0 so it should panic on subtraction with overflow?